### ISSUE TITLE
Make the default database encoding utf8mb4

### ISSUE THREAD

**Issue by tburry** [vanilla/vanilla#5228]
Depends on #5227 and a utility/update on infrastructure first.

Change the following settings in config-defaults:

```php
$Configuration['Database']['CharacterEncoding'] = 'utf8mb4';
$Configuration['Database']['CharacterEncoding'] = 'utf8mb4_unicode_ci';
```

**Comment by DaazKu** [comment]
See: https://github.com/vanilla/vanilla/issues/4309#issuecomment-268059018

**Comment by DaazKu** [comment]
Everything should be fine (DigitalSpy is already on ut8mb4) but it would be nice to have someone else test this out.
